etc/security/mh_default_org.xml

```diff
@@ -450,6 +442,8 @@
 
         <!-- Uncomment the line below to support JWT.
         <ref bean="jwtHeaderFilter" /> -->
+        <!-- Additionally/alternatively uncomment this to support passing a JWT in a URL parameter.
+        <ref bean="jwtQueryParameterFilter" /> -->
```

Comment:
I didn't even consider having another composite filter for that position. Thanks for the idea! :D

I pushed another commit restructuring this a bit further. I left the OAuth/LTI filter where it is now because I wasn't sure how to classify it. You think this is more correct now? :thinking: